fornax-hea/build-sas.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/bash
+
+# Build script to setup a conda environment for XMM's SAS toolkit - based on the 
+#  build-ciao.sh script and the SciServer XMMSAS dockerfile 
+#  (https://github.com/sciserver/sciserver-compute-images/blob/master/heasarc/xmmsas/Dockerfile)
+
+
+########### Validation and setup of directories ###########
+# Location of support data
+if [ -z $SUPPORT_DATA_DIR ]; then
+    echo "ERROR: SUPPORT_DATA_DIR not defined"
+    exit 1
+fi
+
+# Sets up the working directory where SAS will assembled
+WORKDIR=/tmp/sas
+mkdir -p $WORKDIR
+cd $WORKDIR
+
+# Deletes any existing contents of the working directory
+rm -rf * > /dev/null 2>&1
+###########################################################
+
+
+############# Definition of software versions #############
+ubuntu_version=24.04
+sas_version=22.1.0
+py_version=3.12
```

Comment:
Is this a requirement for sas or just a choice?
If the latter, the images have PYTHON_VERSION variable defined, so use that instead